custom/enikshay/case_utils.py

```diff
@@ -154,9 +154,7 @@ def get_all_episode_cases_from_person(domain, person_case_id):
     occurrence_cases = get_all_occurrence_cases_from_person(domain, person_case_id)
     for occurrence_case in occurrence_cases:
         all_cases = case_accessor.get_reverse_indexed_cases([occurrence_case.get_id])
-        episode_cases += [case for case in all_cases
-                          if case.type == CASE_TYPE_EPISODE and
-                          case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
+        episode_cases += [case for case in all_cases if case.type == CASE_TYPE_EPISODE]
```

Comment:
it looks like this is used in more than just the referral report, are these changes ok for that as well